src/ResourceManager/Advisor/Commands.Advisor/help/Disable-AzureRmAdvisorRecommendation.md

```diff
@@ -0,0 +1,175 @@
+zureRMAdvisorRecommendation
+
+SuppressionId : daf24e78-af2d-e8d3-9c50-fa970edc2937
+Ttl           : -1
+Id            : /subscriptions/{user_subscription}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/xyz/providers/Microsoft.Advisor/recommendations
+                /{recommendation_id}/suppressions/HardCodedSupressionName
+Name          : HardCodedSupressionName
+Type          : Microsoft.Advisor/suppressions
+```
+
+Creating a suppression by piping two cmdlet. First cmdlet will return one recommendation and second cmdlet will create a suppression for the same.
+
+## PARAMETERS
+
+### -Days
+Days to disable.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+The powershell object type PsAzureAdvisorResourceRecommendationBase returned by Get-AzureRmAdvisorRecommendation call.
+
+```yaml
+Type: PsAzureAdvisorResourceRecommendationBase
+Parameter Sets: InputObjectParameterSet
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -RecommendationName
+ResourceName of the recommendation
+
+```yaml
+Type: String
+Parameter Sets: NameParameterSet
+Aliases: Name
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceId
+ResourceID of the recommendation to be suppressed (space delimitited).
+
+```yaml
+Type: String
+Parameter Sets: IdParameterSet
+Aliases: Id
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### System.String
+
+### Microsoft.Azure.Commands.Advisor.Cmdlets.Models.PsAzureAdvisorResourceRecommendationBase
+
+## OUTPUTS
+
+### System.Collections.Generic.List`1[[Microsoft.Azure.Commands.Advisor.Cmdlets.Models.PsAzureAdvisorSuppressionContract, Microsoft.Azure.Commands.Advisor, Version=0.1.0.0, Culture=neutral, PublicKeyToken=null]]
```

Comment:
Please manually fix these outputs to be singular - the generator does not update these.